sdk/core/azure-core/tests/test_utils.py

```diff
@@ -1,42 +0,0 @@
-# ------------------------------------
```

Comment:
@zhzhh97 I think this file might have accidentally gotten deleted when you rebased? Can we add it back?